pyrit/orchestrator/prompt_sending_orchestrator.py

```diff
@@ -9,24 +9,28 @@
 from pyrit.models.prompt_request_piece import PromptDataType
 from pyrit.models.prompt_request_response import PromptRequestResponse
 from pyrit.orchestrator import Orchestrator
+from pyrit.orchestrator.scoring_orchestrator import ScoringOrchestrator
 from pyrit.prompt_normalizer import PromptNormalizer
 from pyrit.prompt_normalizer.normalizer_request import NormalizerRequest
 from pyrit.prompt_target import PromptTarget
 from pyrit.prompt_converter import PromptConverter
+from pyrit.score import Scorer
+
 
 logger = logging.getLogger(__name__)
 
 
 class PromptSendingOrchestrator(Orchestrator):
     """
     This orchestrator takes a set of prompts, converts them using the list of PromptConverters,
-    and sends them to a target.
+    scores them with the provided scorer, and sends them to a target.
```

Comment:
```suggestion
    optionally scores them with the provided scorers, and sends them to a target.
```
How about that?

Wait, actually the scoring comes after the target, doesn't it?

Reply:
Fixed wording